docs_2.0/manual_EN/4.ngql-guide/12.vertex-statements/3.upsert-vertex.md

```diff
@@ -0,0 +1,50 @@
+# UPSERT VERTEX
+
+```nGQL
+UPSERT VERTEX <vid> SET <update_columns> [WHEN <condition>] [YIELD <columns>]
+```
+
+- `vid` is the ID of the vertex to be updated.
+- `update_columns` is the properties of the vertex to be updated. For example, `tag1.col1 = $^.tag2.col2 + 1` means to update `tag1.col1` to `tag2.col2+1`.
+
+    **NOTE:**  `$^` indicates the vertex to be updated.
+
+- `condition` is some constraints. Only when the conditions are met, `UPSERT` is executed successfully. `condition` supports expression operations.
+- `columns` is the columns to be returned, `YIELD` returns the latest updated values.
+
+`UPSERT` is a combination of `UPDATE` and `INSERT`. Use `UPSERT VERTEX` to update properties on a vertex or insert a new vertex. The `UPDATE VERTEX` statement only updates one tag of a vertex at a time.
```

Comment:
"The `UPDATE VERTEX` statement only updates one tag of a vertex at a time." ==> Why emphasize this here? "Like UPDATE VERTEX, the UPSERT VERTEX statement only updates one tag of a vertex at a time?"